ffmpeg/ffmpeg_test.go

```diff
@@ -1705,6 +1705,37 @@ func TestTranscoder_IgnoreUnknown(t *testing.T) {
 	run(cmd)
 }
 
+func TestTranscoder_FramesSingle(t *testing.T) {
+	wd, err := os.Getwd()
+	if err != nil {
```

Comment:
Minor, but `require.NoError(t, err)` only takes one line and produces slightly more readable output

Reply:
Fixed